mutters-core/src/main/java/com/rabidgremlin/mutters/core/NoIntentMatch.java

```diff
@@ -0,0 +1,33 @@
+/* Licensed under Apache-2.0 */
+package com.rabidgremlin.mutters.core;
+
+import java.util.HashMap;
+
+/**
+ * An IntentMatch used to indicate no good match was found by an IntentMatcher.
+ * 
+ * @author rabidgremlin
+ *
+ */
+public class NoIntentMatch extends IntentMatch
+{
+  /**
+   * Creates a NoIntentMatch with empty matcher scores.
+   * 
+   */
+  public NoIntentMatch()
+  {
+    super(Intent.NONE, new HashMap<Slot, SlotMatch>(), null, new MatcherScores());
+  }
+
+  /**
+   * Creates a NoIntentMatch with the specified matcher scores. Allows further
+   * processing to be done based on the scores even if no match was found.
+   * 
+   * @param matcherScores The matcher scores.
+   */
+  public NoIntentMatch(MatcherScores matcherScores)
```

Comment:
would be nice if it could also take ```String utterance```

Reply:
I think it should always take ```utterance``` and ```matcherScores```

Reply:
mmm, let me investigate this one

Reply:
Have made utterance required but not matcher scores. These default to empty scores if not passed.